apps/backend/src/lib/emails.tsx

```diff
@@ -221,9 +222,30 @@ export async function sendEmailWithKnownErrorTypes(options: SendEmailOptions): P
   }
 }
 
+export async function sendEmailWithoutRetries(options: SendEmailOptions): Promise<Result<undefined, {
+  rawError: any,
+  errorType: string,
+  canRetry: boolean,
+  message?: string,
+}>> {
+  const res = await _sendEmailWithoutRetries(options);
+  await prismaClient.sentEmail.create({
+    data: {
+      tenancyId: options.tenancyId,
```

Comment:
don't forget to log `userId` too

Reply:
will do that later; userId isn't always guaranteed